discord/ext/commands/context.py

```diff
@@ -54,6 +54,10 @@ class Context(discord.abc.Messageable):
         :func:`on_command_error` event then this dict could be incomplete.
     prefix: :class:`str`
         The prefix that was used to invoke the command.
+    clean_prefix: :class:`str`
```

Comment:
This shouldn't be in this section and should be merged into the property docstring